src/handleAction.js

```diff
@@ -12,7 +12,9 @@ export default function handleAction(type, reducers, defaultState) {
     : [reducers.next, reducers.throw];
 
   if (nextReducer === undefined || throwReducer === undefined) {
-    throw TypeError("reducers argument must be either a function or an object of shape {next, throw}");
+    throw new TypeError(
+      'reducers argument must be either a function or an object of shape {next, throw}'
```

Comment:
>reducers must be a reducer function or an object with shape {next, throw} where 'next' and 'throw' are reducer functions

I know you were broken with ESLint for long lines, so you could use string concatenation:

```js
throw new TypeError(
  'reducers must be a reducer function or an object with ' + 
  'shape {next, throw} where 'next' and 'throw' are reducer functions'
)
```